multi-node/aws/pkg/config/config.go

```diff
@@ -511,7 +508,7 @@ func cidrOverlap(a, b *net.IPNet) bool {
 	return a.Contains(b.IP) || b.Contains(a.IP)
 }
 
-func withTrailingDot(s string) string {
+func WithTrailingDot(s string) string {
```

Comment:
Nit: At some point we should probably move these weird little dns-related helper methods to a separate "utils" type lib package. Another example would be `isSubDomain()`. Non-blocker for now.